test/crud-test.js

```diff
@@ -1066,6 +1066,29 @@ describe("CRUD operations", function() {
             });
         });
 
+        it("Insert image does not exist", function(done) {
```

Comment:
Test is great. I just think that implementation should explicitly test for non-existing path.

Reply:
Seen that I will remove the try/catch, I will be able to make test on this throw exception